harness/determined/cli/trial.py

```diff
@@ -436,6 +439,7 @@ def create_json_file_in_dir(content: Any, file_path: str) -> None:
                 "fetch trial logs",
                 [
                     Arg("trial_id", type=int, help="trial ID"),
+                    Arg("--json", action="store_true", help="print JSON"),
```

Comment:
nit: borrow from here https://github.com/hamidzr/determined/blob/56cc437db2a5a402ab051720de7bb82dc0e9f6e6/harness/determined/cli/_util.py#L15

Reply:
Resolved in new commits -- extended this to all output_formatting_args present in `experiment.py`, `task.py`, `trial.py`